### PR TITLE
Some whitespace changes

### LINKED ISSUE #1056: coding style
search and replace

```c++
template<typename>
// ->
template <typename>
```

```c++
template <typename...>
template <typename...args_t>
template <typename... args_t>
template <typename ... args_t>
// ->
template <typename ...>
template <typename ...args_t>
template <typename ...args_t>
template <typename ...args_t>
```

### PR DESCRIPTION
**Pull request by smehringer**
Officially to get down the issue list, unofficially because I am procrastinating...

Resolves #1056

### CODE REVIEW

File: include/seqan3/core/pod_tuple.hpp [hunk 1]
```diff
@@ -346,7 +346,7 @@ constexpr auto const && get(seqan3::pod_tuple<types...> const && t) noexcept
  * \relates seqan3::pod_tuple
  * \see [std::tuple_element](https://en.cppreference.com/w/cpp/utility/tuple/tuple_element)
  */
-template <std::size_t i, template <typename...> typename t, typename ...types >
+template <std::size_t i, template <typename ...> typename t, typename ...types >
```
Comment (eseiler) [inline include/seqan3/core/pod_tuple.hpp]:
```suggestion
template <std::size_t i, template <typename ...> typename t, typename ...types>
```

File: include/seqan3/core/pod_tuple.hpp [hunk 2]
```diff
@@ -359,7 +359,7 @@ struct tuple_element<i, t<types...>>
  * \see std::tuple_size_v
  * \relates seqan3::pod_tuple
  */
-template <template <typename...> typename t, typename ...types >
+template <template <typename ...> typename t, typename ...types >
```
Comment (eseiler) [inline include/seqan3/core/pod_tuple.hpp]:
```suggestion
template <template <typename ...> typename t, typename ...types>
```

File: include/seqan3/core/type_traits/template_inspection.hpp [hunk 3]
```diff
@@ -57,17 +57,17 @@ struct transfer_template_args_onto
  */
 template <template <typename ...> typename source_template,
           template <typename ...> typename target_template,
-          typename ... source_arg_types>
+          typename ...source_arg_types>
 //!\cond
     requires requires ()
     {
-        std::declval<target_template<source_arg_types...>>();
+        std::declval<target_template <source_arg_types...>>();
```
Comment (eseiler) [inline include/seqan3/core/type_traits/template_inspection.hpp]: Are we sure about this?

Comment (smehringer) [reply include/seqan3/core/type_traits/template_inspection.hpp]: nope this is a regex error. thanks for catching this

Comment (smehringer) [inline include/seqan3/core/type_traits/template_inspection.hpp]:
```suggestion
        std::declval<target_template<source_arg_types...>>();
```

**Pull request merged**